cmd/statusd/main.go

```diff
@@ -68,15 +69,27 @@ var (
 	firebaseAuth = flag.String("shh.firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")
 
 	syncAndExit = flag.Int("sync-and-exit", -1, "Timeout in minutes for blockchain sync and exit, zero means no timeout unless sync is finished")
+
+	searchTopics   topicLimitsFlag
```

Comment:
I'd add a comment that these are about discovery. It isn't obvious from their names.